common/cuda_api_wrappers/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.17)
+project(cuda_api_wrappers)
+
+find_package(CUDAToolkit)
+if(NOT TARGET CUDA::cudart OR NOT TARGET CUDA::cuda_driver)
+  message(WARNING cuda runtime library or cuda driver library is not found.)
+  return()
+endif()
+
+include(FetchContent)
+fetchcontent_declare(cuda-api-wrappers
+  GIT_REPOSITORY https://github.com/eyalroz/cuda-api-wrappers.git
+  GIT_TAG 831666a0bfd1af0f44f4fa234ee2d983d347fcaa # v0.6.1
```

Comment:
I think it would be better to import with vcs, but what do you think?

Reply:
I'm fine with moving this vendor package to repos file, but I would not like to increase the number of packages in repos as possible.

Reply:
https://github.com/autowarefoundation/autoware/blob/main/autoware.repos#L28-L69
we currently use vcs to manage external libraries etc., so I think vcs is better than downloading with cmake, what do you think?

Reply:
I don't really like the idea of not being able to build without an internet connection. If possible, I would like to finish the process of requiring a internet connection when the environment is built. (Right now, it also is required at build time for DNN weight file downloads.)